dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -107,6 +114,63 @@ message TopicEventResponse {
   TopicEventResponseStatus status = 1;
 }
 
+// TopicEventBulkRequestEntry represents a single message inside a bulk request
+message TopicEventBulkRequestEntry {
+  // Unique identifier for the message.
+  string entryID = 1;
+
+  // The content of the event.
+  bytes event = 2;
+  
+  // content type of the event contained.
+  string content_type = 3;
```

Comment:
Please refer: https://github.com/dapr/dapr/issues/2218#issuecomment-1241232708.
As per this agreed upon struct(s), cloudEvent / rawPayload can be part of an entry.
In cloudEvent, we already have dataContentType/data_content_type, it may be confusing to have dataContentType/data_content_type at both places.